packages/material-ui-lab/src/Slider/Slider.test.js

```diff
@@ -136,35 +136,35 @@ describe('<Slider />', () => {
     });
 
     it('should render thumb in initial state', () => {
-      const button = wrapper.find('button');
-      assert.strictEqual(button.props().style.left, '0%');
+      const thumb = wrapper.find('div').filterWhere(n => n.hasClass(classes.thumbWrapper));
+      assert.strictEqual(thumb.props().style.transform, 'translateX(0%)');
     });
 
     it('should render tracks in initial state', () => {
       const tracks = wrapper.find('div').filterWhere(n => n.hasClass(classes.track));
       const trackBefore = tracks.at(0);
       const trackAfter = tracks.at(1);
 
-      assert.strictEqual(trackBefore.props().style.width, '0%');
-      assert.strictEqual(trackAfter.props().style.width, 'calc(100% - 5px)');
+      assert.strictEqual(trackBefore.props().style.transform, 'translateY(-50%) scaleX(0)');
```

Comment:
This looks like we are testing implementation details. Our visual regression tests are better suited for this. Especially if we're talking about the initial state.

Reply:
I kind of agree, do you think that I should just remove this test?
What about the others?

Reply:
I would remove everything that is inline styles related. If you think that you need to test positioning beyond what vrtest does with our demos you might add a visual regression test for the slider to `./test/regressions/tests` but I don't think it's necessary right now.

Reply:
I removed the unit tests. I don't think the unit tests as written added more than the visual testing as the demo page have the disabled sliders with multiple values.

What might add some value is to add visual test for every combination of disabled, vertical, rtl with different values that could have cought the current bug with vertical rtl. However if you don't think it's necessary I accept that. Testing has diminishing returns but not diminishing maintenance cost. If you change your mind and do want visual tests for those kinds of scenarios, I can of course add them.